core/src/protect.c

```diff
@@ -149,12 +149,18 @@ Result expose_wallet(EncryptedMasterSeed *encrypted_master_seed,
     return Result_EXPOSE_WALLET_UNEXPECTED_MASTER_SEED_LEN_FAILURE;
   }
 
+  // __CAUTION__EXPOSE_MASTER_SEED needs to be deliberately defined at build
```

Comment:
Great thought. How about we just delete this statement. I understand it was for debugging and we don't necessarily need it now ?

Reply:
Keeping the code that prints out master seed bytes (under the appropriate circumstances) can still be helpful for debugging. Because I'm aware of at least one such use case in the past, I'm leaning towards leaving the code here, and making it difficult to misuse.